src/Compiler/Service/service.fs

```diff
@@ -274,6 +274,9 @@ type BackgroundCompiler
     /// creates an incremental builder used by the command line compiler.
     let CreateOneIncrementalBuilder (options: FSharpProjectOptions, userOpName) =
         node {
+            use _ =
+                Activity.Start "BackgroundCompiler.CreateOneIncrementalBuilder" [| "project", options.ProjectFileName |]
```

Comment:
Similarly here and above - should we use the same tag name for all things that are a project name/path? Or should we use the names of parameters in a given function? I don't mind either way.

Reply:
For reading produced traces, I prefer having names as consistent (e.g. always "project") as possible.